tools/run-tests.py

```diff
@@ -284,6 +284,7 @@ def create_binary(job, options):
         subprocess.check_output(build_cmd)
         ret = 0
     except subprocess.CalledProcessError as err:
+        print(err.output)
```

Comment:
I feel it is very bad that you cannot see any output on build fail. This line fixes this.